documentation/modules/upgrading/con-upgrade-paths.adoc

```diff
@@ -6,31 +6,31 @@
 = Strimzi upgrade paths
 
 [role="_abstract"]
-Two Strimzi upgrade paths are possible.
+Two upgrade paths are available for Strimzi.
 
 Incremental upgrade::
-Upgrading Strimzi from the previous minor version to version {ProductVersion}.
+An incremental upgrade involves upgrading Strimzi from the previous minor version to version {ProductVersion}.
 
 Multi-version upgrade::
-Upgrading Strimzi from an old version to version {ProductVersion} within a single upgrade (skipping one or more intermediate versions).
-+
-For example, upgrading from Strimzi 0.24.0 directly to Strimzi {ProductVersion}.
-+
-NOTE: A multi-version Strimzi upgrade might still support the current version of a Kafka deployment.
+A multi-version upgrade involves upgrading an older version of Strimzi to version {ProductVersion} within a single upgrade, skipping one or more intermediate versions. 
+For example, upgrading directly from Strimzi 0.30.0 to Strimzi {ProductVersion} is possible. 
 
 [id='con-upgrade-paths-kafka-versions-{context}']
-== Supported Kafka versions
+== Support for Kafka versions when upgrading
+
+When upgrading Strimzi, it is important to ensure compatibility with the Kafka version being used.
 
-Decide which Kafka version to upgrade to before starting the Strimzi upgrade process.
+Multi-version upgrades are possible even if the supported Kafka versions differ between the old and new versions. 
+However, if you attempt to upgrade to a new Strimzi version that does not support the current Kafka version, xref:con-upgrade-cluster-operator-unsupported-kafka-str[an error indicating that the Kafka version is not supported is generated]. 
+In this case, you can upgrade the Kafka version as part of the Strimzi upgrade by changing the `spec.kafka.version` in the `Kafka` custom resource to the supported version for the new Strimzi version.
```

Comment:
I think `can` is too soft here? Would `have to` or `must` be better choice? At the end, there is no other option.

```suggestion
In this case, you have to upgrade the Kafka version as part of the Strimzi upgrade by changing the `spec.kafka.version` in the `Kafka` custom resource to the supported version for the new Strimzi version.
```